lib/mirage.ml

```diff
@@ -729,13 +729,13 @@ let ipv4_qubes db ethernet arp = ipv4_qubes_conf $ db $ ethernet $ arp
 
 let ipv6_conf ?addresses ?netmasks ?gateways () = impl @@ object
     inherit base_configurable
-    method ty = ethernet @-> time @-> mclock @-> ipv6
+    method ty = ethernet @-> random @-> time @-> mclock @-> ipv6
     method name = Name.create "ipv6" ~prefix:"ipv6"
     method module_name = "Ipv6.Make"
     method packages = Key.pure [ package ~sublibs:["ipv6"] "tcpip" ]
```

Comment:
it is not clear to me what exactly `1` means.  mirage does not depend on tcpip.  tcpip does not depend on mirage.   Instead, the unikernels generated by mirage configure should be constrained to tcpip = 3.0.0.  But since mirage3.0.0 is already out, this cannot be done (easily).

with `3`, we should be careful which interfaces/devices need these constraints.  Functoria's `package` mechanism can deal with lower and upper bounds, and merges thereof.